### PR TITLE
Added myst-nb

### PR DESCRIPTION
**Pull request by anirrudh**
Adding myst-nb again, resolving some weird issues. 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by anirrudh** [comment]
@scopatz @CJ-Wright can you please merge? :+1:

File: recipes/myst-nb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - pyyaml
+    - sphinx >=2,<4
+    - sphinx-togglebutton >=0.2.2
+
+test:
+  imports:
+    - myst_nb
+    - myst_nb.nb_glue
+  requires:
+    - pip
```
Comment (CJ-Wright) [inline recipes/myst-nb/meta.yaml:41]: ?

Comment (scopatz) [reply recipes/myst-nb/meta.yaml]: I removed those lines if you want to take another look

**Comment by scopatz** [comment]
Ready for review @conda-forge/staged-recipes

File: recipes/myst-nb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% set name = "myst-nb" %}
+{% set version = "0.9.2" %}
```
Comment (isuruf) [inline recipes/myst-nb/meta.yaml:2]: 0.10.0 is in PyPI with the license file

Comment (scopatz) [reply recipes/myst-nb/meta.yaml:2]: CC @anirrudh

Comment (anirrudh) [reply recipes/myst-nb/meta.yaml:2]: thanks @isuruf @scopatz  - shall update!

**Comment by anirrudh** [comment]
@chrisjsewell added you as a maintainer as well :)

File: recipes/myst-nb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+about:
+  home: https://github.com/executablebooks/myst_nb
+  summary: A Jupyter Notebook Sphinx reader built on top of the MyST markdown parser.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - anirrudh
+    - chrisjsewell
```
Comment (scopatz) [inline recipes/myst-nb/meta.yaml:51]: @chrisjsewell please confirm that you are willing to be a maintainer (so there is a record here)

Comment (chrisjsewell) [reply recipes/myst-nb/meta.yaml:51]: Can we just close this and use my PR.

Comment (scopatz) [reply recipes/myst-nb/meta.yaml:51]: Which ever one works first, I'll merge

Comment (chrisjsewell) [reply recipes/myst-nb/meta.yaml:51]: They will both work at the same time, because they are both waiting for jupyter-cache to be updated

**Comment by anirrudh** [comment]
Closing as #12511 was merged

**Pull request closed without merging**